examples/gatsby-wordpress-gutenberg-default-example/frontend/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "gatsby-wordpress-gutenberg-default-example",
+  "private": true,
+  "description": "A simple example to get up and developing quickly with Gatsby",
+  "version": "0.1.0",
+  "dependencies": {
```

Comment:
Shouldn't this include our plugin as dependency? I think it should work with workspaces, so it resolves to the local one. Once we published it though, it is a valid example for projects using the published one.

Reply:
Yes definitilely, I missed this.

Reply:
- [ ] Add dependecy